FWCore/Integration/test/BuildFile.xml

```diff
@@ -440,6 +440,19 @@
     <use name="FWCore/MessageLogger"/>
     <use name="clhep"/>
   </library>
+  <library file="TransformIntProducer.cc, TransformIntStreamProducer.cc" name = "FWCoreIntegrationTransformIntProducer">
+    <flags EDM_PLUGIN="1"/>
+    <use name="FWCore/Framework"/>
+    <use name="FWCore/ParameterSet"/>
+  </library>
+  <test name="testFWCoreIntegrationTransform" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py"/>
+  <test name="testFWCoreIntegrationTransform_onPath" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --onPath"/>
+  <test name="testFWCoreIntegrationTransform_noTransform" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --noTransform"/>
+  <test name="testFWCoreIntegrationTransform_noTransform_onPath" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --noTransform --onPath"/>
+  <test name="testFWCoreIntegrationTransform_stream" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --stream --onPath"/>
+  <test name="testFWCoreIntegrationTransform_stream_onPath" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --stream --onPath"/>
+
```

Comment:
Would `--noTransform --stream` and `--noTransform --stream --onPath` test cases add any value?

Reply:
> Would `--noTransform --stream` and `--noTransform --stream --onPath` test cases add any value?

Unless I missed something, this question was not yet answered. I don't have further comments, so we could move towards integrating this PR into `master` (i.e. remove the "RFC") as an experimental feature.

Reply:
I don't think so. I think `--noTransform` is really just to test the framework behavior directly (which is the same for the global and the stream form of the module), not with the internals of the modules.

Reply:
Ok. (and no need to follow up later).